src/components/toast/toast.spec.js

```diff
@@ -249,9 +263,9 @@ describe("Toast", () => {
           wrapper = mount(
             <Toast open onDismiss={onDismiss} disableAutoFocus />
           );
-          const icon = wrapper.find("[data-element='close']").first();
-
-          expect(icon).not.toBeFocused();
+          const toast = wrapper.find(ToastStyle);
```

Comment:
Is this actually testing what the test says it is testing? The ToastStyle seems to be the styled component for the Toast itself but the Test wants to make sure close icon isn't focussed. Or are we able to safely assume that if there's no tabindex set on Toast Message then the close button/icon wouldn't be focussed?